### PR TITLE
User image upload function

### PR DESCRIPTION
**Pull request by dylancherry98**
Hi Team,

I finished implementing a user upload feature for new listings (equipment for rent).

Uploaded images, as well as seeded images seem to be displaying properly on all necessary pages.

Please review!

### CODE REVIEW

File: app/views/listings/index.html.erb [hunk 1]
```diff
@@ -10,7 +10,11 @@
     <% @listings.each_with_index do |listing, index| %>
       <%= link_to listing_path(listing) do %>
         <div class="card-listing">
+        <% if listing.image.attached? %>
+          <%= cl_image_tag listing.image.key%>
+        <% else %>
           <img src=<%=listing.photo%>>
```
Comment (nafriedman) [inline app/views/listings/index.html.erb:16]: change this to an image_tag please 🙏

File: app/views/listings/my_listings.html.erb [hunk 2]
```diff
@@ -5,7 +5,11 @@
       <% @listings.each do |listing| %>
         <%= link_to listing_path(listing) do %>
           <div class="card-listing">
+          <% if listing.image.attached? %>
+            <%= cl_image_tag listing.image.key %>
+          <% else %>
             <img src=<%=listing.photo%>>
```
Comment (nafriedman) [inline app/views/listings/my_listings.html.erb:11]: same thing here

File: app/views/listings/show.html.erb [hunk 3]
```diff
@@ -2,7 +2,11 @@
   <h1 class="margin-top"><%=@listing.name%></h1>
 
   <div class="margin-top d-flex align-items-center">
+  <% if @listing.image.attached? %>
+    <%= cl_image_tag @listing.image.key, width:400 %>
+  <% else %>s
     <img src=<%=@listing.photo%> width="400px">
```
Comment (nafriedman) [inline app/views/listings/show.html.erb:8]: and here

Comment (dylancherry98) [reply app/views/listings/show.html.erb]: I changed both!

**Comment by nafriedman** [comment]
Looks awesome 💯

**Pull request merged**